packages/components/notification/src/composables/use-actions.ts

```diff
@@ -0,0 +1,93 @@
+import { type Ref, computed, ref } from 'vue'
+import { type MaybeComputedRef, resolveUnref as toValue } from '@vueuse/core'
+import { type Mutable, debugWarn, keysOf } from '@element-plus/utils'
+import type { ButtonProps } from '@element-plus/components/button'
+import type { NotificationAction, NotificationProps } from '../notification'
```

Comment:
To keep the code style consistent, we'd better separate the variable introduction and type introduction and put the type introduction at the bottom.